core/util.go

```diff
@@ -340,3 +340,18 @@ func (s *SliceSet[T]) Append(element T) {
 	}
 	*s = append(*s, element)
 }
+
+// ImportFromHost is a hack to import data from a specified host directory into
+// buildkit - this is useful if we already have the content on the host - and
+// need to get it into buildkit somehow.
+func ImportFromHost(bk *buildkit.Client, path string, includePatterns []string, opts ...llb.ConstraintsOpt) llb.State {
```

Comment:
extreme nit: `ImportFromEngineHost` is slightly more clear since `Host` refers to the client's Host in other parts of the code